packages/react-juce/src/lib/Backend.ts

```diff
@@ -32,6 +104,12 @@ export class ViewInstance {
   constructor(id: string, type: string, props?: any, parent?: ViewInstance) {
     this._id = id;
     this._type = type;
+    this._border = {
```

Comment:
No need to carry this state on the ViewInstance, let's have `parseBorderProp` return a value which looks like this

Reply:
(Sorry for delay returning to this)

I could be not understanding something here, but don't we need to carry state around in the ViewInstance? There's no longer necessarily a single border property, but potentially a series of border- properties that may cumulatively specify what actually needs to be drawn. Surely we need to accumulate that state somewhere?

Reply:
Hmm. Guess we could have parseBorder(Side)Prop() take a border object and return a transmuted one. Still need to have something accumulating the values, though.